numpy/_globals.py

```diff
@@ -52,11 +52,21 @@ class VisibleDeprecationWarning(UserWarning):
     """
     pass
 
-
-class _NoValue(object):
+class _NoValueType(object):
     """Special keyword value.
 
-    This class may be used as the default value assigned to a deprecated
-    keyword in order to check if it has been given a user defined value.
+    The instance of this class may be used as the default value assigned to a
+    deprecated keyword in order to check if it has been given a user defined
+    value.
     """
-    pass
+    __instance = None
+    def __new__(cls):
+        # ensure that only one instance exists
+        if not cls.__instance:
+            cls.__instance = super(_NoValueType, cls).__new__(cls)
+        return cls.__instance
+
+    def __repr__(self):
+        return "<no value>"
```

Comment:
Taking the advice from [@seberg's comment](https://github.com/numpy/numpy/pull/9592#issuecomment-324244145) to deliberately obfuscate how to use this value, focusing just on the docs repr.